### PR TITLE
fix: Update span ops

### PR DESCRIPTION
**Pull request by cleptric**
To unblock the work on Performance Issues, we have to update some span ops.
https://github.com/getsentry/sentry/discussions/37083

### CODE REVIEW

File: tests/Tracing/Twig/TwigTracingExtensionTest.php [hunk 1]
```diff
@@ -58,7 +58,7 @@ public function testEnter(Profile $profile, string $spanDescription): void
 
         $this->assertCount(2, $spans);
         $this->assertNull($spans[1]->getEndTimestamp());
-        $this->assertSame('view.render', $spans[1]->getOp());
+        $this->assertSame('symfony.view', $spans[1]->getOp());
```
Comment (AbhiPrasad) [inline tests/Tracing/Twig/TwigTracingExtensionTest.php:61]: Let’s not change this yet - it’s still up for debate

Comment (cleptric) [reply tests/Tracing/Twig/TwigTracingExtensionTest.php:61]: https://develop.sentry.dev/sdk/performance/span-operations/#web-server
There is also `template.render`

Comment (AbhiPrasad) [reply tests/Tracing/Twig/TwigTracingExtensionTest.php]: Yeah - we’ll make decision about this next week and then we can just update it then.

**Review by AbhiPrasad — APPROVED** [review]

**Review by k-fish — APPROVED** [review]
Looks good!

**Comment by ste93cry** [comment]
I chose the names of these operations because they are what the OpenTelemetry standard uses. Wouldn't it make more sense to support them in Sentry rather than shifting even more from a common standard which is well established in the industry?

**Comment by AbhiPrasad** [comment]
OpenTelemetry doesn’t have the concept of a span op like we do, so fundamentally we’ll be diverging here anyway. Mapping otel span name -> description works for us, but span operation we should be free to work with.

We use the `db` prefix to generate performance issues: https://blog.sentry.io/2022/09/19/performance-issues-slow-you-can-act-on-quickly/

We are working on getting closer to the standard in general though. In https://github.com/getsentry/develop/pull/686 I mapped out the transformation between opentelemetry <-> to sentry data, and we’re working on improving this.

**Pull request merged**